Add timeline documentation (#2544)
OctopusDeploy/docs: src/pages/docs/releases/deployment-timeline.md

---
layout: src/layouts/Default.astro
pubDate: 2024-10-28
modDate: 2024-10-28
title: Deployment Timeline

> description: View and interact with the deployment Timeline

Reviewer: ```suggestion
description: View and interact with the Deployment Timeline
```

navOrder: 132
---

The Deployment Timeline provides visual representation of all deployments made to an [Environment](docs/infrastructure/environments) for a [Project](/docs/projects) within a [Tenant](docs/projects/tenants). In additional to providing an easy to understand view of deployment data, the Deployment Timeline allows easy access to perform common workflows, such as deploying the last successful/stable deployment into your environment.

> It can be access by clicking the **Timeline** button within the release details page.

Reviewer: ```suggestion
It can be access by clicking the **Timeline** button within the deployment details page.
```

:::figure
![The Deployment Timeline button](/docs/releases/timeline/timeline-button.png)
:::

> A drawer opens showing all deployments related to the release you are viewing, sorted by when the deployment was created, 

Reviewer: ```suggestion
A drawer opens showing all deployments for the current Environment, Project and Tenant you are viewing, sorted by when the deployment was created.
```
